ray-operator/controllers/ray/utils/util.go

```diff
@@ -68,6 +68,24 @@ func IsRunningAndReady(pod *corev1.Pod) bool {
 	return false
 }
 
+func CheckRouteName(s string, n string) string {
+	// 63 - (max(8,6) + 5 ) - length of namespace name + 1 => 49 - namespace name
+	// 6 to 8 char are consumed at the end with "-head-" or -worker- + 5 generated.
```

Comment:
Routes are not generated for worker nodes, are they?

Reply:
True; updated!